frontend/src/views/web/challenge/delete-challenge/delete-challenge.html

```diff
@@ -0,0 +1,23 @@
+<section class="text-center pd-20 width-500">
+    <div class="row ht-wt-100p">
+        <div class="col s12 m12">
+            <div class="ev-card-body">
+                <form name="deleteChallengeForm" ng-submit="challenge.deleteChallenge(deleteChallengeForm.$valid)">
+                    <div class="pass-title">Delete Challenge</div>
+                    <div class="input-field align-left">
+                        <input placeholder="Enter challenge name" type="text" ng-model="nameInput">
```

Comment:
Please change the text with `Please type in the name of the challenge to confirm.`